scripts/deploy/btc-bridge.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+source "$(dirname "$0")/deploy_functions.sh"
+
+IC_NETWORK="local"
+BITCOIN_NETWORK="regtest"
+
+function usage() {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  -h, --help                                      Display this help message"
+  echo "  -b, --bitcoin-network <network>                 Bitcoin network (regtest, testnet, mainnet) (default: regtest)"
+  echo "  -e, --evm-principal <principal>                 EVM Principal"
+  echo "  -i, --ic-network <network>                      Internet Computer network (local, ic) (default: local)"
+  echo "  -m, --install-mode <mode>                       Install mode (create, init, reinstall, upgrade)"
+  echo "  --ckbtc-minter <canister-id>                    CK-BTC minter canister ID"
+  echo "  --ckbtc-ledger <canister-id>                    CK-BTC ledger canister ID"
+}
+
+ARGS=$(getopt -o e:b:i:m:h --long evm-principal,ic-network,install-mode,bitcoin-network,ckbtc-ledger,ckbtc-minter,help -- "$@")
+while true; do
+  case "$1" in
+    -e|--evm-principal)
+      EVM_PRINCIPAL="$2"
+      shift 2
+      ;;
+
+    -i|--ic-network)
+      IC_NETWORK="$2"
+      shift 2
+      ;;
+
+    -b|--bitcoin-network)
+      BITCOIN_NETWORK="$2"
+      shift 2
+      ;;
+
+    -m|--install-mode)
+      INSTALL_MODE="$2"
+      shift 2
+      ;;
+
+    --ckbtc-minter)
+      CKBTC_MINTER="$2"
+      shift 2
+      ;;
+    
+    --ckbtc-ledger)
+      CKBTC_LEDGER="$2"
+      shift 2
+      ;;
+
+    -h|--help)
+      usage
+      exit 255
+      ;;
+
+    --)
+      shift
+      break
+      ;;
+
+    *)
+      break
+  esac
+done
+
+assert_isset_param "$INSTALL_MODE" "INSTALL_MODE"
+assert_isset_param "$BITCOIN_NETWORK" "BITCOIN_NETWORK"
+if [ "$IC_NETWORK" != "local" ]; then
+  assert_isset_param "$EVM_PRINCIPAL" "EVM_PRINCIPAL"
+  assert_isset_param "$CKBTC_MINTER" "CKBTC_MINTER"
+  assert_isset_param "$CKBTC_LEDGER" "CKBTC_LEDGER"
+fi
+
+LOG_SETTINGS="opt record { enable_console=false; in_memory_records=opt 10000; log_filter=opt \"info,evm_core=debug,evm=debug\"; }"
+OWNER=$(dfx identity get-principal)
+SIGNING_STRATEGY="variant { ManagementCanister = record { key_id = variant { Dfx }; } }"
```

Comment:
Here and for other canisters, for local dfx deployment we should use key_id `Dfx`, and for IC deployment  - `Production`. I don't think we need to use the local signing strategy (with PK) at all.